src/common/utility_win.cpp

```diff
@@ -55,29 +55,8 @@ const QString systemThemesC()
 
 namespace OCC {
 
-void Utility::setupFavLink(const QString &folder)
+void Utility::setupFavLink(const QString &)
```

Comment:
Perhaps we should mark this as `deprecated`? I mean the Linux version has been broken for several years as well (not saying that that one should necessarily be discarded, but so far it just didn't do anything anyway).